charts/buildbuddy/templates/deployment.yaml

```diff
@@ -3,10 +3,7 @@ kind: Deployment
 metadata:
   name: {{ include "buildbuddy.fullname" . }}
   labels:
-    app.kubernetes.io/name: {{ include "buildbuddy.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    helm.sh/chart: {{ include "buildbuddy.chart" . }}
+    {{- include "buildbuddy.labels" . | nindent 4 }}
     {{- if .Values.extraPodLabels }}
     {{- .Values.extraPodLabels | toYaml | nindent 4 }}
     {{- end }}
```

Comment:
I think previously, we did not differentiate between Deployment object's label and Pod's label. And because of that, we used `extraPodLabels` for both.

Now that we have a separate `extraLabels`, I think it might make more sense to remove `extraPodLabels` from the Deployment object's label and make it dedicated to the BuildBuddy pod only. WDYT?
```suggestion
```

Reply:
sounds good to me